### DISCUSSION TITLE
How to get boundary conditions and loads from cdb file

### DISCUSSION THREAD

**Discussion by Saivellampati** [pyansys/pymapdl-reader#459] [General]
I want to know how to get BCs and loads from .cdb file. I am able to get them from .rst file. Please let me know if it is possible to get them from cdb file

**Comment by RobPasMue** [comment]
@germa89 @mikerife - some help here?

**Comment by germa89** [comment]
@Saivellampati I think that `CDB` files are basically a bunch of MAPDL commands that replicates a model. Therefore, to get the model, you need to execute the commands, hence you will need PyMAPDL and MAPDL. 

`pymapdl-reader` can read RST files, but it is not intended for manipulating CDB files.
